src/main.rs

```diff
@@ -28,6 +30,34 @@ fn main() {
     //   a. Open & parse twim-config toml file, and for each project:
     //   b. Find & update (or add) the entry
     //   c. Write the file to disk
+    let mut twim_config_file = File::open(TWIM_CONFIG_PATH)
+        .expect("Unable to open twim config file");
+    template = String::new();
+    twim_config_file.read_to_string(&mut template)
+        .expect("Unable to read twim config file");
+    let mut twim_config: twim_config::Config = toml::from_str(&template)
+        .expect("Unable to parse twim-config");
+    
+    let mut projects_matched = 0;
+    for twim_project in &mut twim_config.projects {
+        for bot in &projects.bots {
+            if bot.title == twim_project.title {
+                println!("Found {} in data and twim-config", twim_project.title);
+                projects_matched = projects_matched + 1;
+                twim_project.title = "nope".to_string();
+            }
+        }
+        for bridge in &projects.bridges {
+            if bridge.title == twim_project.title {
+                println!("Found {} in data and twim-config", twim_project.title);
+                projects_matched = projects_matched + 1;
+            }
+        }
+    }
```

Comment:
Is there a more "rust-ish" (and efficient?) way to do it? Is there a way to do it for all the types in twim_project that wouldn't require copy/pasting hose for loops?